core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -64,6 +67,8 @@
 
   private static final boolean PACKET_STREAMING_ENABLED =
       Configuration.getBoolean(PropertyKey.USER_PACKET_STREAMING_ENABLED);
+  private static final boolean PASSIVE_CACHE_DISABLED =
```

Comment:
can we avoid using xxx_disabled but always xxx_enabled? also apply the same comment for the property name (and its value)